v2/pkg/config/load.go

```diff
@@ -62,6 +91,26 @@ func LoadConfig(data []byte) (c v1alpha2.ImageSetConfiguration, err error) {
 	return c, nil
 }
 
+// LoadConfigDelete loads data into a v1alpha2.ImageSetConfiguration instance
+func LoadConfigDelete(data []byte) (c v1alpha2.DeleteImageSetConfiguration, err error) {
```

Comment:
Is it possible to do any kind of reuse of LoadConfig since the code is the same?

Reply:
Yes good suggestion , I used generics